src/MeadeCommandProcessor.cpp

```diff
@@ -816,22 +816,30 @@ bool gpsAqcuisitionComplete(int &indicator);  // defined in c72_menuHA_GPS.hpp
 //      Returns:
 //        "HHMMSS"
 //
-// :XGh#
+// :XGhN#
 //      Description:
-//        Get Heater values
+//        Get value for heaters
 //      Information:
-//        Get the current values for both heaters
+//        Get value for heater N in the range from 0-10, or both heaters if no N is given.
 //      Returns:
-//        "NNN,NNN"
+//        "NNN,NNN#" or "N#"
 //
 // :XShNnnn#
 //      Description:
 //        Set strength for heater N to nnn
 //      Information:
-//        Set PWM duty cycle for heater N (0 or 1) to nnn (0 - 10). The final PWM value is goes from 0 to DEW_HEATER_N_MAX.
+//        Set scale of the PWM duty cycle for heater N (0 or 1) to nnn (0 - 10). The final PWM value goes from 0 to DEW_HEATER_N_MAX.
 //      Returns:
 //        1#
 //
+// :XRST#
+//      Description:
+//        Restart the mount
+//      Information:
+//        Restarts the mount like into a state which is like the one you get when you use the reset button.
```

Comment:
Too many 'like's. Let's do `Restarts the mount in the same way pressing the Reset button on the LCD would.`